RecoTauTag/HLTProducers/src/PFJetsTauCorrelationCondition.cc

```diff
@@ -0,0 +1,86 @@
+#include "RecoTauTag/HLTProducers/interface/PFJetsTauCorrelationCondition.h"
+#include "Math/GenVector/VectorUtil.h"
+#include "DataFormats/HLTReco/interface/TriggerTypeDefs.h"
+#include "FWCore/Utilities/interface/EDMException.h"
+#include "DataFormats/TauReco/interface/PFTau.h"
+#include "DataFormats/Math/interface/deltaR.h"
+
+//
+// class declaration
+//
+PFJetsTauCorrelationCondition::PFJetsTauCorrelationCondition(const edm::ParameterSet& iConfig)
+    : tauSrc_(consumes<trigger::TriggerFilterObjectWithRefs>(iConfig.getParameter<edm::InputTag>("TauSrc"))),
+      pfJetSrc_(consumes<reco::PFJetCollection>(iConfig.getParameter<edm::InputTag>("PFJetSrc"))),
```

Comment:
It is not necessary to specify type in `consumes`, i.e. `consumes<...>(...)`->`consumes(...)`